Task2/Kuba/NoughtsAndCrossesWPF/NoughtsAndCrossesWPF/ViewModel/ViewModel.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
+
+namespace NoughtsAndCrossesWPF
+{
+    public class ViewModel //I'm not sure if I have put correct methods in this class. I feel like it's a bit messed but have no better idea how to organize this.
+    {
+        public string[] translatedOneDimensionalBoard = new string[GameBoard.boardSize * GameBoard.boardSize];
+        private ObservableCollection<string> _ObservableBoard;
+        public ObservableCollection<string> ObservableBoard { get { return _ObservableBoard; } }
+        static WPFBoardCommunication communicator = new WPFBoardCommunication();
+        GameBoard board = new GameBoard(); 
+        Game game = new Game(communicator);
```

Comment:
keeping Game and GameBoard separate is a bit confusing, Game should probably have Board inside